dev/bots/flutter_gold_status.dart

```diff
@@ -0,0 +1,90 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io' as io;
+
+import 'package:path/path.dart' as path;
+import 'package:process/process.dart';
+
+import 'utils.dart';
+
+final String flutterRoot = path.dirname(path.dirname(path.dirname(path.fromUri(io.Platform.script))));
+final String cirrusPR = io.Platform.environment['CIRRUS_PR'];
+
+/// Used to check for any tryjob results from Flutter Gold associated with a PR.
+Future<void> main() async {
+  // Allow time for the results to have finished processing on Gold's end
+  print('$clock Waiting 30 seconds to start status check...');
+  await Future<void>.delayed(const Duration(seconds: 30));
+  try {
+    print('Running task: Flutter Gold Tryjob Status Check');
+    print('═' * 80);
+    await _queryTryjobStatus();
+  } on ExitException catch (error) {
+    error.apply();
+  }
+  print('$clock ${bold}Tryjob status check complete, no untriaged results.$reset');
+}
+
+Future<void> _queryTryjobStatus() async {
+  final String currentCommit = await _getCurrentCommit();
+  final Uri requestForTryjobStatus = Uri.parse(
+    'http://flutter-gold.skia.org/json/changelist/github/$cirrusPR/$currentCommit/untriaged'
+  );
+  String rawResponse;
+  bool needsTriage = true;
+
+  print('$clock ${green}Requesting:$reset\n$requestForTryjobStatus');
+
+  // Continue checking until there are no untriaged digests.
+  // TODO(Piinks): This should also have the opportunity to fail when new images
+  //  have been marked negative. This may be possible after
+  //  https://bugs.chromium.org/p/skia/issues/detail?id=9783 is done.
+  while (needsTriage) {
+    try {
+      final io.HttpClient httpClient = io.HttpClient();
+      final io.HttpClientRequest request = await httpClient.getUrl(requestForTryjobStatus);
+      final io.HttpClientResponse response = await request.close();
+
+      rawResponse = await utf8.decodeStream(response);
+      final Map<String, dynamic> decodedResponse = json.decode(rawResponse) as Map<String, dynamic>;
+
+      if (decodedResponse['digests'] == null) {
+        needsTriage = false;
+      } else {
+        print('${red}Tryjob generated new images.$reset\n'
```

Comment:
It would be ideal if we made this ping the issue the first time it happens, so people who are tagged/subscribed to the issue will know to do something.  Otherwise, this is just an in progress check that will fail after an hour and you'll need to read the logs.  We'd need to have a GitHub oauth token to do it - we could use the same one Flutter GitHub bot uses and it would get posted as that user.

Reply:
I had thought about doing that, but Gold is actually implementing comments on their side right now: https://bugs.chromium.org/p/skia/issues/detail?id=9006 I wasn't sure it was worth the time implementing a second commenting feature on our side.